Update to GDPR analytics article (#3963)
PostHog/posthog.com: contents/blog/best-gdpr-compliant-analytics-tools.md

---

> date: 2022-08-11

Author: ```suggestion
date: 2022-08-12
```

author: ["joe-martin", "andy-vandervell"]
featuredImage: ../images/blog/posthog-company-culture-blog.png
featuredImageType: full
categories: ["Guides", "Privacy"]
---

> The GDPR places significant restrictions on how you can use tools like Google Analytics to track and collect user data.

Reviewer: ```suggestion
GDPR places significant restrictions on how you can use tools such as [Google Analytics](https://isgoogleanalyticsillegal.com/) to track and collect user data.
```
